### DISCUSSION TITLE
Add Instances from All Predictions on Current Video

### DISCUSSION THREAD

**Discussion by Toshea111** [talmolab/sleap#1061] [Ideas]
It would be useful if there was an option to 'Add Instances from All Predictions on Current Video' to supplement the existing 'Add Instances from All Predictions on Current Frame'.

I have found that when using predictions to generate additional labels, it would be easier to check through each frame, and then add all new predictions once corrections were made, rather than adding them frame-by-frame.

**Comment by Kagiro-K** [comment]
This would be incredibly helpful! Are there any plans for this in the next update for SLEAP? This will allow users to include predicted frames in the next batch of training without having to individually click on them in each frame.
